tests/post/features/ingress.feature

```diff
@@ -19,12 +19,36 @@ Feature: Ingress
         When we perform an HTTP request on port 80 on a control-plane IP
         Then the server should not respond
 
+    Scenario: Failover of Control Plane Ingress VIP using MetalLB
+        Given the Kubernetes API is available
+        And we are on a multi node cluster
+        And MetalLB is enabled
+        When we stop the node hosting the Control Plane Ingress VIP
+        Then the node hosting the Control Plane Ingress VIP changed
+        And we are able to login to Dex as 'admin@metalk8s.invalid' using password 'password'
+
     Scenario: Change Control Plane Ingress IP to node-1 IP
         Given the Kubernetes API is available
         And we are on a multi node cluster
         And pods with label 'app.kubernetes.io/name=ingress-nginx' are 'Ready'
-        When we update control plane ingress IP to node 'node-1' IP
+        When we disable MetalLB and set control plane ingress IP to node 'node-1' IP
```

Comment:
Let's try not to do two actions in the same `when`. Also, I think this should be a `given`, in this case:

```gherkin
Given MetalLB is disabled
...
When we set control plane ingress IP to node 'node-1' IP
...
```

Reply:
To me "given" was more "given environment is in this specific state" (basically checking the state of the environment but do not change anything) and "when" more like "when we do this specific action" ... "then we are in this state"
It's not right ?

The split is a bit difficult because you do not want to disable MetalLB and keep the ingress IP to a ... non-existing VIP. To me it's a single action "I switch from MetalLB to non-MetalLB using Node-1 IP" (maybe I should reword it a bit :thinking: )